src/steadystateproblem.cpp

```diff
@@ -225,11 +226,15 @@ bool SteadystateProblem::initializeBackwardProblem(Solver *solver,
     }
 
     /* Will need to write quadratures: set to 0 */
+    xQ.reset();
     xQB.reset();
+    xQBdot.reset();
```

Comment:
I originally wanted to avoid introducing this function... Turned out, we need it...

Reply:
not sure what you mean by this comment.